install/linux/usr/share/odemis/sim/secom2-sem-ctrl-sim.odm.yaml

```diff
@@ -0,0 +1,221 @@
+SECOMv2: {
+    class: Microscope,
+    role: secom,
+    children: ["Light Engine", "Optical Objective", "Optical Emission Filter",
```

Comment:
Can you group them by HW relation? Like all the SEM components, stage, optical light components, detector. 
You mentioned to me that you usually follow the order of the beam path - could you do this here as well? So SEM, stage, light optics, detector?